config/routes.rb

```diff
@@ -46,6 +46,10 @@
     get "/have-you-been-evicted", to: "have_you_been_evicted#show"
     post "/have-you-been-evicted", to: "have_you_been_evicted#submit"
 
+    # Question: Have you got somewhere to live?
```

Comment:
I'd suggest moving this up, so the questions remain in the same order as in the Figma prototype (https://www.figma.com/file/tibZIxmeYW0Z3qEL4CEF72/Triage-tool---prototypes?node-id=953%3A2)

Reply:
Done, most of these seem to be out of order so I'll raise a PR after this to change them round